### PR TITLE
Remove promotion logic from s3_index

### PR DESCRIPTION
**Pull request by MarcoPolo**
@gabriel

### CODE REVIEW

File: packaging/prerelease/s3_index.sh [hunk 1]
```diff
@@ -41,9 +41,3 @@ fi
 
 echo "Checking if we need to promote a release for testing ($platform)"
 "$release_bin" promote-test-releases --bucket-name="$bucket_name" --platform="$platform"
```
Comment (MarcoPolo) [inline packaging/prerelease/s3_index.sh]: Didn't feel like adding the code on the slackbot side, but if you want I can do that too.

Comment (gabriel) [inline packaging/prerelease/s3_index.sh]: Nope... leave this here... I will probably nuke this as some point and have every build auto promote to testing.

**Comment by gabriel** [comment]
lg2m

**Comment by oconnor663** [comment]
I think this PR caused https://keybase.atlassian.net/browse/CORE-3077.

**Pull request merged**